dataService.js

```diff
@@ -0,0 +1,42 @@
+// import required json arrays
+const allCategories = require('./src/data/categories.json').categories
+const allProducts = require('./src/data/products.json').products
+
+/*
+ * Map individual product objects to product id attributes
+ */
+function getProducts (product) {
+  productMap[product.id] = product
+}
+
+/*
+ * Map categoryName as a value for attribute for categoryId
+ */
+function getCategories (category) {
+  categoryMap[category.id] = category.categoryName
```

Comment:
See step 4. Create the entire `categoryMap` here, using `forEach` on `allCategories`. Shouldn't need to take a parameter.